cpp/CMakeLists.txt

```diff
@@ -533,6 +533,9 @@ set(ARROW_TESTING_PC_LIBS "")
 set(ARROW_TESTING_PC_REQUIRES "")
 
 # For parquet.pc.
+set(PARQUET_PC_CFLAGS "")
+set(PARQUET_PC_CFLAGS_PRIVATE " -DPARQUET_STATIC")
```

Comment:
Is this a hack? Or `Cflags.private` requires only static linking?

Reply:
This is not a hack.
This is a normal `.pc` usage.

> Or `Cflags.private` requires only static linking?

Right.
`pkg-config --cflags parquet` uses only `Cflags` and
`pkg-config --cflags --static parquet` uses both of `Cflags` and `Cflags.private`.